### PR TITLE
Add suffixes to typos from "j" to "l"

### PR DESCRIPTION
**Pull request by int-y1**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -30992,6 +30992,10 @@ jurnaler->journaler
 jurnales->journals
 jurnaling->journaling
 jurnals->journals
+jurney->journey
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:30995]: `gurney`/`gurneys` too?

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -31377,7 +31414,12 @@ laungage->language
 laungages->languages
 launguage->language
 launguages->languages
+launh->launch
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:31417]: Perhaps `laugh` (and friends below)?

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -31759,7 +31822,9 @@ linez->lines
 lingth->length
 linheight->lineheight
 Linix->Linux
-linke->linked, like,
+linke->linked, link, like, line,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:31825]: Add `links` to the list? ad `minke` as in "minke whale"?

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -32001,6 +32109,7 @@ lsip->lisp
 lsit->list, slit, sit,
 lsits->lists, slits, sits,
 luckly->luckily
+luetooth->bluetooth
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:32112]: Perhaps **B**luetooth, as it is a brand name?

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:31825]: I'll add `links`. I couldn't find `minke`.

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:32112]: There were other entries with `bluetooth`, so I capitalized them too.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**